Update guest-code-101.md (#1605)
risc0/risc0: website/api/zkvm/guest-code-101.md

- For a guide to writing and running your first guest code, check out our [zkVM
  Quick Start][quickstart].
- For a simple example, check out the [Hello World tutorial][hello-world], where

>   the [guest] receives one input from the [host] and commits the result of its operation to

Reviewer: ```suggestion
  the [guest] receives an input from the [host], and then commits a value to 
```
